### ISSUE TITLE
A question about the loss mutation during network training

### ISSUE THREAD

**Issue by Beyond-AA** [NVlabs/stylegan2-ada-pytorch#191]
Hello, I did not use transfer learning, and retrained a network. During my training, the loss of the D network suddenly increased at 4800kimg, and the loss of the G network suddenly decreased. The value of FID dropped rapidly from 12 o’clock. Can you explain to me the reason for the sudden change?
![1635920621(1)](https://user-images.githubusercontent.com/67589128/140017740-8a6ea0dd-47b4-4aa8-872d-cd0af89b73c3.jpg)
![1635920661(1)](https://user-images.githubusercontent.com/67589128/140017760-a8ea4f0c-a1f7-492a-aec4-2f718b410344.jpg)
![1635920693(1)](https://user-images.githubusercontent.com/67589128/140017785-b41dd9be-000e-4ef9-ad8a-de5ea828f277.jpg)
